typings/index.d.ts

```diff
@@ -0,0 +1,152 @@
+// Typescript Declaration by
+// Taylor Stapleton on November 6th 2018
+// For the project simple-youtube-api
+// Project: https://github.com/HyperCoder2975/simple-youtube-api
+// Writer of this file: https://github.com/OscarXcore
+
+declare module 'simple-youtube-api' {
+  class YouTube {
+    key: string;
+    request: Request;
+
+    constructor(key: string);
+    public getVideo(url: string, options?: object): Promise<Video | Error>;
+    public getPlaylist(url: string, options?: object): Promise<Playlist | Error>;
+    public getChannel(url: string, options?: object): Promise<Channel | Error>;
+    public getVideoByID(id: string, options?: object): Promise<Video | Error>;
+    public getPlaylistByID(id: string, options?: object): Promise<Playlist | Error>;
+    public getChannelByID(id: string, options?: object): Promise<Channel | Error>;
```

Comment:
All of the above return types are wrong. These methods will never resolve with an error.

Reply:
So they resolve nothing then if they dont?

Reply:
I need something specific. I dont think it liked `Promise<?Video>` so much. Pretty sure typescript had a fit because that is a JSDoc type and was like "no"